lib/pure/os.nim

```diff
@@ -1224,12 +1232,17 @@ proc fileNewer*(a, b: string): bool {.rtl, extern: "nos$1", noNimScript.} =
 
 proc getCurrentDir*(): string {.rtl, extern: "nos$1", tags: [], noNimScript.} =
   ## Returns the `current working directory`:idx:.
+  ## This is similar to Linux, Mac, BSD, Windows PowerShell ``pwd`` terminal command.
+  ## This is the path where the Nim-compiled binary is *run*.
+  ## Its not available for JavaScript and NimScript backends, nor at compile-time.
+  ## Its available for C, C++ and ObjectiveC backends.
```

Comment:
Similar advice: only mention a backend if a proc's behavior is different or nonexistent on it. Thus, mention that this proc doesn't work on JavaScript and NimScript, and leave it at that.